src/solvers/refinement/string_refinement.cpp

```diff
@@ -1184,28 +1188,6 @@ bool string_refinementt::check_axioms()
             << " universal string axioms can be violated" << eom;
     debug() << violated_not_contains.size()
             << " not_contains string axioms can be violated" << eom;
-
-    if(use_counter_example)
```

Comment:
I know `use_counter_example` is always false for the moment but I'd like to keep this code there to see if it can influence the performances. Maybe we should have the possibility to set this as a command line option and have tests for it, but we don't want to multiply command line options especially when they don't make sense to the user, so I don't mind this going away, given that performances of the solver are not our biggest issue at the moment.